src/algorithms/DouglasRachford.jl

```diff
@@ -38,9 +38,23 @@ maxit(sol::DRSIterator) = sol.maxit
 
 converged(sol::DRSIterator, it) = blockmaxabs(sol.FPR_x)/sol.gamma <= sol.tol
 
-verbose(sol::DRSIterator, it) = sol.verbose > 0
+verbose(sol::DRSIterator)     = sol.verbose > 0
+verbose(sol::DRSIterator, it) = sol.verbose > 0 && (sol.verbose == 1 ? true : (it == 1 || it%100 == 0))
```

Comment:
Related to the above comment: should be `sol.verbose == 2 ? true`. Furthermore, I would make that `100` an option (`output_freq` maybe? Of course defaulting to `100`).